erpnext/manufacturing/doctype/job_card/job_card.py

```diff
@@ -21,8 +21,9 @@ def validate_time_logs(self):
 
 		if self.get('time_logs'):
 			for d in self.get('time_logs'):
-				if get_datetime(d.from_time) > get_datetime(d.to_time):
-					frappe.throw(_("Row {0}: From time must be less than to time").format(d.idx))
+				if d.from_time and d.to_time: 
+					if get_datetime(d.from_time) > get_datetime(d.to_time):
+						frappe.throw(_("Row {0}: From time must be less than to time").format(d.idx))
```

Comment:
```suggestion
				if d.from_time and d.to_time and get_datetime(d.from_time) > get_datetime(d.to_time):
					frappe.throw(_("Row {0}: From time must be less than to time").format(d.idx))
```

Reply:
- Any reason for marking this as resolved without making the changes ?

Reply:
My bad, mistakenly I click on the resolution option.